x/mint/keeper.go

```diff
@@ -38,9 +38,13 @@ func NewKeeper(cdc *codec.Codec, key sdk.StoreKey, sk staking.Keeper, bk bank.Ke
 
 // Mint credits {coin} to the {recipient} account, and reflects the increase in issuance
 func (k Keeper) Mint(ctx sdk.Context, recipient sdk.AccAddress, coin sdk.Coin) (err sdk.Error) {
-	_, _, err = k.bk.AddCoins(ctx, recipient, sdk.Coins{coin})
-	if err != nil {
-		return err
+
+	// recipient could be empty in case minting to validator outstanding rewards
```

Comment:
Does `AddCoins` throw an error if the recipient is empty?

Reply:
For oracle vote claim, we move from direct reward, to validator's account address, to validator's outstanding reward pool to let share reward with its delegator. In case, giving reward to pool there is no recipient address so we need minting funciton without recipient.